sentinel/daemon/Gemfile

```diff
@@ -0,0 +1,35 @@
+source (ENV['RUBYGEMS_MIRROR'] or 'https://rubygems.org')
+
+ruby '2.1.7' # prevents hard-to-diagnose errors with bundle install
```

Comment:
We should remove these from Gemfiles. The comment here (from the Sentinel source) is false. These lines actually cause more issues than they solve.